### PR TITLE
Update site with new date and venue

### PR DESCRIPTION
**Pull request by benjaoming**
I think this looks good, but I made a lot of typos while doing this update (am in a train) :see_no_evil:

### CODE REVIEW

**Comment by benjaoming** [comment]
Pretty sure that the automatic deployment won't work since GitHub replaced their host SSH keys some months ago.

File: content/cfp.md [hunk 1]
```diff
@@ -44,7 +44,8 @@ We are looking for:
 
 **We'll announce a new deadline soon**
 
-Submit before **March 1st 2022 23:59:59 UTC+2**. The final program will be announced on April 1st (perhaps with a joke).
+Submit before **September 10th 2023 23:59:59 UTC+2**.
+The final program will be announced on September 1st.
```
Comment (valberg) [inline content/cfp.md]: This is an error right? We want the submissions before we announce the final program right?

Comment (benjaoming) [reply content/cfp.md]: Oh yes definitely, is August 10th good?

**Review by jvzammit — APPROVED** [review]
LGTM

File: content/cfp.md (hunk 1 above)
Comment (benjaoming) [reply content/cfp.md]: I picked August 21st so a bit more time to emerge from holidays and a bit of buffer to postpone the deadline if necessary :+1:

**Pull request merged**